src/reporting/reportConversionResults.ts

```diff
@@ -93,3 +94,28 @@ const logMissingPlugins = (plugins: Set<string>, logger: Logger) => {
             .join(""),
     );
 };
+
+const logNotices = (converted: Map<string, ESLintRuleOptions>, logger: Logger) => {
+    const rulesWithNotices = Array.from(converted.values()).filter(
+        ruleOptions => ruleOptions.notices && ruleOptions.notices.length >= 1,
+    );
+
+    if (rulesWithNotices.length > 0) {
+        logger.stdout.write(chalk.yellowBright(`📢 ${rulesWithNotices.length} ESLint`));
+        logger.stdout.write(
+            chalk.yellowBright(rulesWithNotices.length == 1 ? ` rule behaves` : ` rules behave`),
+        );
+        logger.stdout.write(
+            chalk.yellowBright(` differently from their TSLint counterparts: 📢${EOL}`),
+        );
+
+        rulesWithNotices.forEach(rule => {
```

Comment:
There might be a more idiomatic way to write this nested for loop :)

Reply:
Yeah I would move the `logger.stdout.write` to inside the `if (rule.notices !== undefined) {` block. There's not really a need to print out that line.

However, if you change `ruleOptions =>` to something like `(ruleOptions): ruleOptions is RuleWithNotices`, where that's a type that says `.notices` isn't `undefined`, you can skip the `if` statement altogether. Let's go with that to simplify the output JS code. https://fettblog.eu/typescript-type-predicates

Reply:
Nice! Thanks!